setup.cfg

```diff
@@ -53,7 +53,7 @@ include =
 [flake8]
 max-line-length = 88
 exclude = build/*, dist/*, ansq.egg-info/*, .tox/*, .venv/*, .git/*, .eggs/*
-extend-ignore = E203  # E203 conflicts with PEP8; see https://github.com/psf/black#slices
```

Comment:
FYI  Inline comments are illegal, see a flake8 exception:

```
ValueError: Error code '#' supplied to 'extend-ignore' option does not match '^[A-Z]{1,3}[0-9]{0,3}$'
```

If you put the comment above the `tend-ignore`, then `setup-cfg-fmt` would strip it. Okay then, there is nothing we can do a guess.